docs/spec.md

```diff
@@ -4173,7 +4173,8 @@ where:
 <!-- markdownlint-disable line-length -->
 * `padded_inputs = pad(inputs[:], init_values[:], padding[:, 0], padding[:, 1], base_dilations[:] - 1)`.
 * `window_start = result_index * window_strides`.
-* `windows = slice(padded_inputs[:], window_start, window_start + window_dimensions, window_dilations)`.
+* `window_end = clamp(0, window_start + window_dimensions * window_dilations, shape(inputs[0]))`.
```

Comment:
Two comments 
1. Before applying the `window_dimension` to the padded input, it can be grown using `window_dimaltion`, by `window_dilation -1`. That is `window_start =  window_start + window_dimensions  +  window_dilations - 1`. Can you explain why are we multiplying? 
2. Should we consider `padded_input`'s dimensions for clamping? Not sure why using `shape(inputs[0])`  ?

Reply:
1. Yes you're right, should be adding not multiplying.
2. `window_start` contain the index relative to the `paddedInput`. Similarly, `window_end` should contain the index relative to the `padded_input`. The `SliceOp` here would be slicing a window defined by its start index (`window_start`) and limit index (`window_end`) out of the `padded_input`.

Reply:
Re: On 2. Should we consider padded_input's dimensions for clamping? Not sure why using shape(inputs[0]) ?

>>Similarly, window_end should contain the index relative to the padded_input. 

In that case,  why not clamp it using the limits of `padded_input`?

Also as per slice semantics, `limit_indices >= start_indices`, so the clamp should be

`window_end = clamp(window_start, window_start + window_dimensions  + window_dilations - 1, shape(inputs[0]))`, right?

Reply:
I see what you mean, yes `window_end` should be (corrected typo on the shape):
`window_end = clamp(window_start, window_start + window_dimensions  + window_dilations - 1, shape(padded_inputs[0]))`

The max clamping part was reflected in the implementation but not the spec, which is updated now along with the min clamping on both.

Reply:
The PR description still mentions multiplication. Let's update it as well?

Reply:
Thanks, updated the description.